backup.sh

```diff
@@ -39,9 +41,18 @@ backup() {
   fi
 
   export PGPASSWORD=$POSTGRES_PASSWORD
-  cmd="pg_dump --host=\"$POSTGRES_HOST\" --port=\"$POSTGRES_PORT\" $cmd_auth_part $cmd_db_part | gzip > $BACKUP_DIR/$archive_name"
-  echo "starting to backup PostGRES host=$POSTGRES_HOST port=$POSTGRES_PORT"
 
+  if [[ ! -z $POSTGRES_DB ]]; then
```

Comment:
I will support this use case in the coming weeks, thanks for open the Pull request, I'm taking care of this image and i will update to pg 15